testdata/testdata.go

```diff
@@ -0,0 +1,11 @@
+package testdata
```

Comment:
I think this use of `testdata` for a Go package is non-idiomatic and could be confusing since `testdata` is treated specially by the Go tool: https://golang.org/cmd/go/#hdr-Package_lists - "Directory and file names that begin with "." or "_" are ignored by the go tool, as are directories named "testdata"."  If you need to have a package for testing, maybe you could give it some other name, like `testpkg` or anything else?  I would expect to see text, JSON, yaml or files in other input formats in the `textdata` directory - or at least would keep the directory available for such in the future.

Reply:
Thank you! I've changed the directory name to `datafortest`. I've also make subdirectory `binary` and `yaml`.

Reply:
I think we should use `testdata` directory and below under package directory rather than root directory. I set new comment  https://github.com/screwdriver-cd/sd-cmd/pull/13#discussion_r172399314